lib/qwerty/source.rb

```diff
@@ -0,0 +1,12 @@
+module Qwerty
+  class Source < Ruote::Participant
+    def on_workitem
+      workitem.fields['source'] = {
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.